### DISCUSSION TITLE
[REQUEST]Item Stacks

### DISCUSSION THREAD

**Discussion by MERRYACS** [Zrips/CMI#2617] [Feature Requests]
Request:
- allow specific amount of item stacks

For example, potion can be stacks with 3, bucket stacks with 18 or more.

Reason:
There are two plugins in the Minecraft plugin market that support ItemStacks, but they all have some bugs. However, their downloads are very high, so I recommend CMI to do the same. Because some servers need to control the item stacks, such as the RPG server, as well as mini games, you can make many unexpected gameplay through ItemStack.

**Comment by mrfloris** [comment]
Hey,

Maybe I misunderstand what you want, but I believe the `cmi more` command (run from console, don't give the players the permission so they can't abuse this (cmi.command.more.oversize)) might be close to what you want.

`cmi more (playerName) (-clone/[amount])`

Maybe you're asking how to do this from a shop, where the player doesn't have the item in their inventory yet, so you can't stack what they're holding?
